### ISSUE TITLE
Updates due on newgem.rubyforge.org

### ISSUE THREAD

**Issue by svoop** [drnic/newgem#1]
Great gem, however, the prose on newgem.rubyforge.org is slightly out of date. In the "Setup" section, you still refer to "config/hoe.rb" for configuration, but that's in the "Rakefile" now.

**Comment by drnic** [comment]
Yeah, probably a few website + README updates due with 1.5.0 release (soon)
